cmds/vpdbootmanager/addentry/add.go

```diff
@@ -0,0 +1,45 @@
+package addentry
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/vpd"
+)
+
+const usage = `Usage:
+boot add [netboot [dhcpv6|dhcpv4] [MAC] | localboot [grub|path [Device GUID] [Kernel Path]]]
+
+Use -h for flag options
+`
+
+// Cmd executes the command line parsing for the add command
+func Cmd() {
+	if len(os.Args) < 3 {
+		fmt.Println(usage)
+		os.Exit(1)
+	}
+	switch os.Args[2] {
+	case "netboot":
+		addNetbootEntry()
+	case "localboot":
+
+	}
+}
+
+func addBootEntry(data []byte) error {
+	for i := 1; i < 10; i++ {
+		key := fmt.Sprintf("Boot%04d", i)
+		if _, err := vpd.Get(key, true); err != nil {
+			if os.IsNotExist(err) {
+				if err := vpd.Set(key, data, true); err != nil {
```

Comment:
I think we want boot variables to be in the RW partition, so `false` instead of `true` here. cc @zaolin to double-check